Functionality_Tests/BaseTest.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Functionality_Tests_Suit
 {
-    public abstract class BaseTest
+    public class BaseTest
     {
         public IWebDriver driver;
```

Comment:
Should be private and readonly. Rename according to codestyles after it :)
Name of fields:
private fields _lowerCaseWithUnderScore 
public or protected fields UpperCase

Reply:
Done protected as well since it is used in SeleniumTests also

Reply:
exactly, great!